client/blocks/login/login-form.jsx

```diff
@@ -448,6 +448,37 @@ export class LoginForm extends Component {
 		);
 	}
 
+	renderMagicLoginLink() {
+		if (
+			! canDoMagicLogin(
+				this.props.twoFactorAuthType,
+				this.props.oauth2Client,
+				this.props.wccomFrom,
+				this.props.isJetpackWooCommerceFlow
+			)
+		) {
+			return null;
+		}
+
+		// The leading space is NO typo! It's added since the first part of the validation message (originating from the BE) contains no trailing space.
+		return this.props.translate(
+			' Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Would you like to {{magicLoginLink}}receive a login link{{/magicLoginLink}}?' )` *ES Score: 16*